e2e-tests/test/hardhat-apis.test.ts

```diff
@@ -84,3 +88,64 @@ xdescribe("hardhat_impersonateAccount & hardhat_stopImpersonatingAccount", funct
     expect(await provider.getBalance(RichAccounts[0].Account)).to.equal(beforeBalance.sub(0.42));
   });
 });
+
+describe("hardhat_setCode", function () {
+  it.only("Should set code at an address", async function () {
```

Comment:
Please remove the `.only`. I'll make a ticket to ensure this doesn't happen again and use something like https://stackoverflow.com/a/54015366
```suggestion
  it("Should set code at an address", async function () {
```